docker-compose.yaml

```diff
@@ -255,8 +255,6 @@ services:
         }
         airflow_version=$$(AIRFLOW__LOGGING__LOGGING_LEVEL=INFO && gosu airflow airflow version)
         airflow_version_comparable=$$(ver $${airflow_version})
-        min_airflow_version=2.2.0
-        min_airflow_version_comparable=$$(ver $${min_airflow_version})
         if (( airflow_version_comparable < min_airflow_version_comparable )); then
```

Comment:
I guess you don't need this check either. But why it's a problem? Are we running airflow<2.2 anywhere?